### PR TITLE
Kivy baseclass chatscreen

### PR DESCRIPTION
**Pull request by osamacis**
Added a chat screen

### CODE REVIEW

**Comment by PeterSurda** [comment]
Put the "Chat" menu item after the messages, and it its own section. Layout looks ok.

**Comment by osamacis** [comment]
> Put the "Chat" menu item after the messages, and it its own section. Layout looks ok.

done with it

File: src/bitmessagekivy/baseclass/chat.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# pylint: disable=import-error, no-name-in-module, too-few-public-methods, too-many-ancestors
+
+'''
+    Payment/subscription frontend
```
Comment (PeterSurda) [inline src/bitmessagekivy/baseclass/chat.py:4]: change docstring

File: src/bitmessagekivy/baseclass/chat.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+'''
+
+from kivy.uix.screenmanager import Screen
+
+
+class Chat(Screen):
+    """Chat Screen class for kivy Ui"""
+
+    def __init__(self, *args, **kwargs):
```
Comment (PeterSurda) [inline src/bitmessagekivy/baseclass/chat.py:13]: if we just call `super`, we don't need this method

**Review by PeterSurda — CHANGES_REQUESTED** [review]

**Comment by PeterSurda** [comment]
> @PeterSurda
> 
> The words "Chat Screen" and "Chat" bother me.
> 
> **Bitmessage is not a chat application!**
> 
> It should be renamed to something else, such as Discussions.

This is for future functionality. This PR contains initial kivy UI.

**Review by PeterSurda — APPROVED** [review]

**Comment by PeterSurda** [comment]
@redarmyfaction Can you please explain what specifically is the problem with chat? This isn't a protocol change, this is just a different UI. The whitepaper doesn't talk about the UI.

**Pull request merged**